python/smqtk/algorithms/saliency/logit_sal.py

```diff
@@ -0,0 +1,247 @@
+from PIL import Image
+import numpy as np
+import os
+from sklearn.metrics.pairwise import euclidean_distances
+import six
+from smqtk.algorithms.relevancy_index import get_relevancy_index_impls
+from smqtk.algorithms import get_descriptor_generator_impls
+from smqtk.algorithms import SmqtkAlgorithm
+from smqtk.algorithms.saliency import SaliencyBlackbox,ImageSaliencyAugmenter
+from smqtk.representation.data_element.memory_element import DataMemoryElement
+from smqtk.utils import plugin
+
+    
+class Logit_SaliencyBlackbox (SaliencyBlackbox):
+    """
+    Logit_SaliencyBlackbox function yields some floating point scalar value for a given masked base image descriptor element that signifies the difference between the confidence value of the query image and masked image descriptors, used by class implementations of 'ImageSaliencyMapGenerator'.
+    """
+
+    def __init__(self, ADJs, rel_index, T_descr):
+        """
+        :param ADJs: is a set of positive and negative adjudications.
+        :param type: set of (smqtk.representation.DescriptorMemoryElemets)
```

Comment:
I'm guessing what you were going for here was a parameter type specification. This is not correct reStructuredText syntax. What you probably want is:
```
:type ADJs: set[smqtk.representation.DescriptorMemoryElement]
```
The values for types are generally what would be used for [python3 type annotations](https://docs.python.org/3/library/typing.html).

Additionally, type annotation can be included in the ``param`` part if not overly long:
```
:param set[smqtk.representation.DescriptorMemoryElement] ADJs:
    is a set of positive and negative adjudications.
```

Reply:
On a less technical note, the ``ADJs`` description is not really correct as this value is treated as a tuple of two distinct sets later in the code. It may make more sense to separate this out into two parameters: one for positive and one for negative adjudications. This parameters is also implicitly required to be formatted the same as the call to ``rel_index.rank()`` as its simply argument expanded (``*self.ADJs``).

Reply:
Split the input arguments into pos_descriptors and neg_descriptor. Still using self.ADJs but creating it in class __init__() instead of it being an input argument. Also changed get_config to reflect the changes.

Reply:
The first part of this comment was not resolved.

Reply:
Done.